controller/src/main/java/io/pravega/controller/store/stream/tables/HistoryRecord.java

```diff
@@ -25,13 +25,18 @@
  * HistoryRecords are of variable length, so we will maintain markers for
  * start of row and end of row. We need it in both directions because we need to traverse
  * in both directions on the history table
- * Row : [length][List-of-active-segment-numbers, [length-so-far], scaleTime][length]
+ * Row : [length][epoch][List-of-active-segment-numbers], [length-so-far], [scaleTime][length]
  */
 public class HistoryRecord {
```

Comment:
This likely won't be the last time this changes. Perhaps we should add a version number